projects/media-viewer/src/lib/annotations/comment-set/comment/comment.component.ts

```diff
@@ -88,8 +89,15 @@ export class CommentComponent implements OnChanges {
     this.editable = true;
   }
 
+  onCommentChange(updatedComment) {
+    const hasUnsavedChanges = this.originalComment.substring(0, this.COMMENT_CHAR_LIMIT) !==
```

Comment:
would this not be better to check if the length of the comment character changed is above the character length. The reason for this is i believe if the user added words and was still under the char limit, even tho it will save, this condition will still be true for hasUnsavedChanges but in fact it should save.